pkg/ipmi/ioctl.go

```diff
@@ -0,0 +1,43 @@
+// Copyright 2019 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Bits and pieces from asm-generic/ioctl.h
+package ipmi
+
+import (
+	"syscall"
+	"unsafe"
+)
+
+const (
```

Comment:
is this stuff really not defined anywhere?

Reply:
Not that I am aware of.